packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: path_provider_linux
 description: Linux implementation of the path_provider plugin
 repository: https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider_linux
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+path_provider%22
-version: 2.1.3
+version: 2.1.4
```

Comment:
I remember even for @visibleForTesting API updates, we need to bump minor? (I might be wrong)

Reply:
The file that I added the new content to shouldn't be exported. I just wanted to be extra-obvious about it being test-only.